src/main-process/atom-application.js

```diff
@@ -15,6 +15,7 @@ const path = require('path')
 const os = require('os')
 const net = require('net')
 const url = require('url')
+const {promisify} = require('util')
```

Comment:
Oh hey! I didn't realize we had access to `promisify` now. This will come in handy.